v1/examples/LifeEventSETClass/death-registration.json

```diff
@@ -0,0 +1,79 @@
+{
+  "iss": "https://scim.example.com",
+  "iat": 1691637001,
+  "jti": "4d3559ec67504aaba65d40b0363faad8",
+  "toe": 1691637178,
+  "events": {
+    "https://ssf.account.gov.uk/v1/deathRegistration": {
+      "subject": {
+        "name": [
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Rogers",
+                "type": "FamilyName"
+              }
+            ]
+          },
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Smith",
+                "type": "FamilyName"
+              }
+            ],
+            "description": "Otherwise or formerly known as"
+          },
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Molone",
+                "type": "FamilyName"
+              }
+            ],
+            "description": "Name before marriage"
+          }
+        ],
+        "sex": [
+          {
+            "value": "Female"
+          }
+        ],
+        "birthDate": [
+          {
+            "value": "1989-07-06",
+            "description": "May be incorrect"
+          }
+        ],
+        "address": [
+          {
+            "subBuildingName": "Flat 4",
+            "buildingNumber": "5",
+            "streetName": "Mystery Street",
+            "addressLocality": "Coolsville",
+            "postalCode": "M15 7",
+            "addressCountry": "US"
+          }
+        ]
+      },
+      "deathRegistrationID": 123456,
+      "deathDate": {
```

Comment:
From some further discussion with HMPO, we've found two fields that are subtly different
- a qualifier about the death date (e.g.  `Deceased found on`, which is used for partial dates)
- a free format death date, which might be used instead of the date value field and we've got no specific description of what it could be

which we've split into the "qualifier" being the "description", and the free format date being an additional field

Reply:
Ok thanks.

Reply:
We definitely need to find a way to explain this in descriptions of the slots, it's quite subtle and I'm not sure I'd really believe what's in there until we get to look at the live event file (assuming someone allows us to do so!!)